### PR TITLE
Reinstate lines removed in error by BHoMBot

### LINKED ISSUE #464: Reinstate lines removed by BHoMBot
#### Description:

Line defining `seachLine` removed during assembley version update:
https://github.com/BHoM/Test_Toolkit/commit/7e53ad8f4fec8afabbd04681100d477ff7b2b649

#### Steps to reproduce:

See above.

#### Expected behaviour:

Test_Toolkit to compile

#### Test file(s):

By inspection

### PR DESCRIPTION
**Pull request by peterjamesnugent**
### Issues addressed by this PR

 Closes #464 

 

 ### Test files

Test by compiling.

 ### Changelog

- Reinstated lines that have been removed in error, that resulted in the toolkit not compiling

 ### Additional comments

### CODE REVIEW

**Comment by peterjamesnugent** [comment]
@BHoMBot check copyright-compliance
@BHoMBot check project-compliance
@BHoMBot check serialisation
@BHoMBot check null-handling

File: CodeComplianceTest_Engine/Compute/CheckAssemblyInfo.cs [hunk 1]
```diff
@@ -88,7 +87,7 @@ private static TestResult CheckAssemblyFileVersion(this List<string> fileLines,
         {
             string currentAssemblyVersion = Query.FullCurrentAssemblyFileVersion();
 
-[assembly: AssemblyFileVersion("7.1.0.0")]
+            string searchLine = $"[assembly: AssemblyVersion(\"";
```
Comment (FraserGreenroyd) [inline CodeComplianceTest_Engine/Compute/CheckAssemblyInfo.cs:90]:
```suggestion
            string searchLine = $"[assembly: AssemblyFileVersion(\"";
```

File: CodeComplianceTest_Engine/Compute/CheckAssemblyInfo.cs [hunk 2]
```diff
@@ -107,7 +106,7 @@ private static TestResult CheckAssemblyFileVersion(this List<string> fileLines,
 
         private static TestResult CheckAssemblyDescription(this List<string> fileLines, TestResult finalResult, string assemblyInfoPath, string documentationLink, string descriptionUrl)
         {
-[assembly: AssemblyDescription("https://github.com/BHoM/Test_Toolkit")]
+            string searchLine = $"[assembly: AssemblyVersion(\"";
```
Comment (FraserGreenroyd) [inline CodeComplianceTest_Engine/Compute/CheckAssemblyInfo.cs:109]:
```suggestion
            string searchLine = $"[assembly: AssemblyDescription(\"";
```

**Review by FraserGreenroyd — CHANGES_REQUESTED** [review]

**Comment by FraserGreenroyd** [comment]
@BHoMBot check copyright-compliance
@BHoMBot check project-compliance
@BHoMBot check serialisation
@BHoMBot check null-handling

**Review by FraserGreenroyd — APPROVED** [review]
Thanks @peterjamesnugent - good catch!

**Pull request merged**